delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/config/AggregateConfig.scala

```diff
@@ -0,0 +1,34 @@
+package ch.epfl.bluebrain.nexus.delta.service.config
+
+import akka.util.Timeout
+import ch.epfl.bluebrain.nexus.sourcing.SnapshotStrategyConfig
+import ch.epfl.bluebrain.nexus.sourcing.processor.{AggregateProcessorConfig, StopStrategyConfig}
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.FiniteDuration
+
+/**
+  * Aggregate configuration.
+  *
+  * @param stopStrategy               the stop strategy configuration for this aggregate [[EventDefinition]]
+  * @param snapshotStrategy           the snapshot strategy configuration for this aggregate [[EventDefinition]]
+  * @param askTimeout                 timeout for the message exchange with the aggregate actor
+  * @param evaluationMaxDuration      timeout for evaluating commands
+  * @param evaluationExecutionContext the execution context where commands are to be evaluated
+  * @param stashSize                  the maximum size allowed for stashing when evaluating
+  */
+final case class AggregateConfig(
+    stopStrategy: StopStrategyConfig,
+    snapshotStrategy: SnapshotStrategyConfig,
+    askTimeout: Timeout,
+    evaluationMaxDuration: FiniteDuration,
+    evaluationExecutionContext: ExecutionContext,
+    stashSize: Int
+) {
+
+  /**
+    * The underlying processor ''AggregateConfig''
+    */
+  def processor: AggregateProcessorConfig =
```

Comment:
We could have in as subconfiguration as the others

Reply:
That complicates things for pureconfig reader, making it more verbose. I don't think it is worth it.

In the app.conf this is - and should be - on the top level

```
    aggregate {
      snapshot-strategy = ${app.defaults.snapshot-every-500}
      stop-strategy = ${app.defaults.passivate-never}
      ask-timeout = 15 seconds
      evaluation-max-duration = 3 seconds
      stash-size = 10
    }
```

That resembles to the case class

Reply:
You can do it without modifying the conf by slightly modilfying your ConfigReader

Reply:
what's the benefit?

Reply:
It is a little more readable as you have the four fields embedded in the processorConfig so you understand it is not used for something else